diff -Naur a/util/perl/OpenSSL/Glob.pm b/util/perl/OpenSSL/Glob.pm
--- a/util/perl/OpenSSL/Glob.pm	1970-01-01 01:00:00.000000000 +0100
+++ b/util/perl/OpenSSL/Glob.pm	2020-01-13 03:40:28.025459508 +0000
@@ -0,0 +1,21 @@
+package OpenSSL::Glob;
+
+use strict;
+use warnings;
+
+use File::Glob;
+
+use Exporter;
+use vars qw($VERSION @ISA @EXPORT);
+
+$VERSION = '0.1';
+@ISA = qw(Exporter);
+@EXPORT = qw(glob);
+
+sub glob {
+    goto &File::Glob::bsd_glob if $^O ne "VMS";
+    goto &CORE::glob;
+}
+
+1;
+__END__ 
